pkg/commands/git.go

```diff
@@ -314,8 +314,12 @@ func (c *GitCommand) usingGpg() bool {
 }
 
 // Commit commits to git
-func (c *GitCommand) Commit(message string) (*exec.Cmd, error) {
-	command := fmt.Sprintf("git commit -m %s", c.OSCommand.Quote(message))
+func (c *GitCommand) Commit(message string, amend bool) (*exec.Cmd, error) {
+	amendParam := ""
+	if amend {
+		amendParam = "--amend "
```

Comment:
Done.

Reply:
@jesseduffield i did this, but i reverted it back with a space before `--`. Lot's of tests fail because moving space to `Sprintf` causes two spaces between `git commit` and `-m` if there's no amend.